app/views/devise/sessions/splash.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="top p-2">
+    <h2 class='text-light text-center h1'>Welcome</h2>
+</div>
+
+<div class="splash">
+    <div class='d-flex splash-footer p-3 justify-content-around gap-2'>
+      <%= link_to 'Sign In', new_user_session_path, class: 'btn border border-light p-2 text-light col-6' %>
+      <%= link_to 'Sign Up', new_user_registration_path, class: 'btn border border-light text-light col-6 p-2' %>
+    </div>
+</div>
```

Comment:
- [ ]  As previous suggested please provide the name of your app on splash page and it should be in the center of the page as it is designed in the app. 

![image](https://github.com/alicemirigo92/Budget-App/assets/43877689/deb27a0a-26a9-4006-8a52-7f7fe541b797)